single_tech_samples/synapseanalytics/sample2_synapseserverless/scripts/deploy_synapse_artifacts.sh

```diff
@@ -167,7 +186,11 @@ createNotebook "Nb_NYCTaxi_Convert_Parquet_to_Delta"
 az synapse spark session list --workspace-name "${SYNAPSE_WORKSPACE_NAME}" --spark-pool-name "${BIG_DATAPOOL_NAME}"
 createNotebook "Nb_NYCTaxi_Config_Operations_Library"
 az synapse spark session list --workspace-name "${SYNAPSE_WORKSPACE_NAME}" --spark-pool-name "${BIG_DATAPOOL_NAME}"
+createNotebook "Nb_NYCTaxi_Config_Operations_Library_No_Wheel"
```

Comment:
q: is it intentional to keep the notebook without wheel (the one with the pip install command on it) on  this repo? if not, consider deleting the notebook without the wheel package and rename the remaining notebook without the "_No_Wheel" suffix, and this would apply to both config and retention notebooks I guess.
Also another question: what's the purpose of all those spark session listing operations (4)? just curious

Reply:
Answer on comment line 183 - # This line allows the spark pool to be available to attach to the notebooks